src/main/resources/templates/components/navbar.peb

```diff
@@ -5,9 +5,11 @@
         <li><h1>Koil.</h1></li>
     </ul>
     <ul class="is-hidden-mobile" data-test="desktop-navbar">
-        {% if global.isAdmin %}
-            <li><a href="/admin" data-test="navbar-admin-link">Admin Panel</a></li>
-        {% endif %}
```

Comment:
What's the deal with getting rid of this?  The admin panel is away?

Reply:
It's just a UI change. Adding in user images and admin link and user handle made the navbar a bit busy. The admin panel link is still in the aside menu 👍